.github/workflows/wheels-dependencies.sh

```diff
@@ -98,6 +99,62 @@ function build_harfbuzz {
     touch harfbuzz-stamp
 }
 
+function build_libavif {
+    if [ -e libavif-stamp ]; then return; fi
+
+    python3 -m pip install meson ninja
+
+    if [[ "$PLAT" == "x86_64" ]] || [ -n "$SANITIZER" ]; then
+        build_simple nasm 2.16.03 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03
+    fi
+
+    local build_type=MinSizeRel
+    local lto=ON
+
+    local libavif_cmake_flags=()
+
+    if [ -n "$IS_MACOS" ]; then
+        lto=OFF
+        libavif_cmake_flags+=(
+            -DCMAKE_C_FLAGS_MINSIZEREL="-Oz -DNDEBUG -flto " \
+            -DCMAKE_CXX_FLAGS_MINSIZEREL="-Oz -DNDEBUG -flto" \
+            -DCMAKE_SHARED_LINKER_FLAGS_INIT="-Wl,-S,-x,-dead_strip_dylibs" \
+        )
+    else
+        if [[ "$MB_ML_VER" == 2014 ]] && [[ "$PLAT" == "x86_64" ]]; then
+            build_type=Release
+        fi
+        libavif_cmake_flags+=(-DCMAKE_SHARED_LINKER_FLAGS_INIT="-Wl,--strip-all,-z,relro,-z,now")
```

Comment:
If I compare the wheel sizes from https://github.com/python-pillow/Pillow/actions/runs/14773502497?pr=8858,
with a commit where I [remove](https://github.com/radarhere/Pillow/commit/73c2f320ce3f1f1b186ba42bfaa0237a0f9a28a5) this line, I find that https://github.com/radarhere/Pillow/actions/runs/14773542213 is actually 8kb smaller.

If this line is purely about reducing file size, then I don't think it's fulfilling that role.

Reply:
Ah, it's because there is a typo. I'll fix it.

Reply:
It actually had to do with a cmake policy that was disabling LTO on linux. See the build here: https://github.com/fdintino/Pillow/actions/runs/14777836226

Reply:
In the PR description, I've updated the paragraph that begins with "In absolute terms,..." and the last column of the table with the latest wheels and shared library file sizes. You can look at the github diff under the list of edits to see the change.